nautobot/core/templates/inc/custom_fields_panel.html

```diff
@@ -15,7 +15,9 @@
                                 <i class="mdi mdi-close-thick text-danger" title="False"></i>
                             {% elif field.type == 'url' and value %}
                                 <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
-                            {% elif value is not None %}
+                            {% elif field.type == 'multi-select' and value %}
+                                {{ value|join:", " }}
+                            {% elif value is not None and value %}
```

Comment:
Sorry to step in the review process, isn't it cleaner to just show the intention?
```suggestion
                            {% elif value or value == 0 %}
```

Reply:
`value is not None and value` would never catch `value == 0`. So if the need is to explicitly act upon the integer `0`, then I agree (since that is equivalent to `False`).

Thanks @Thetacz, don't ever apologize for contributing. :)